src/test/unit_tests/jest_tests/test_nc_nsfs_bucket_cli.test.js

```diff
@@ -329,6 +338,69 @@ describe('cli create bucket using from_file', () => {
 
 });
 
+describe('cli update bucket', () => {
+    const config_root = path.join(tmp_fs_path, 'config_root_manage_nsfs2');
+    const root_path = path.join(tmp_fs_path, 'root_path_manage_nsfs2/');
```

Comment:
I think it is better to make sure that these paths were not defined in the file (I think it appears in the 'cli delete bucket`). You can simply change the number in the suffix.